### ISSUE TITLE
Enhancement / Add Dark Mode Switcher

### ISSUE THREAD

**Issue by vijeet-shah** [vijeet-shah/react_practice#7]
Dark Mode Switcher: Implement a dark mode switcher that toggles the color scheme of the entire app. Use useEffect to update the theme on app load or preference change.

**Comment by bmsptra24** [comment]
Hi! I'm really interested in this issue and would love to contribute to adding the Dark Mode Switcher

**Comment by bmsptra24** [comment]
Hi! I'm really interested in this issue and would love to contribute to adding the Dark Mode Switcher

**Comment by vijeet-shah** [comment]
> Hi! I'm really interested in this issue and would love to contribute to adding the Dark Mode Switcher

Absolutely, I'm excited about your interest in contributing to the Dark Mode Switcher

**Comment by bmsptra24** [comment]
I've got some fantastic news to share to you! 🎉 I've gone ahead and implemented the dark mode feature. Feel free to check out my pull request and let me know what you think! 🚀🔥
